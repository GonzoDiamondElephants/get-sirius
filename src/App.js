import React from 'react';
import LoginButton from './components/loginButton.js';

function App() {
  return (
    <>
      <h1>Home Page</h1>
      <LoginButton />
    </>
  );
}

export default App;
