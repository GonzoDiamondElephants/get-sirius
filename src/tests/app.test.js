import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App.js';

describe('Happy path of App.js', () => {
  it('It successfully check using snapshot', () => {
    let currentHTML = renderer.create(<App />).toJSON();
    expect(currentHTML).toMatchSnapshot();
  });
});
