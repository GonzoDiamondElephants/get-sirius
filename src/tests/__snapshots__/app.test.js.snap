// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Happy path of App.js It successfully check using snapshot 1`] = `
<div>
  Loading.......
</div>
`;
